message.js

```diff
@@ -212,6 +219,7 @@ jspb.Message.ASSUME_LOCAL_ARRAYS =
  *     reduces the size of serialized protos. The price is an extra iteration of
  *     the proto before serialization. This is enabled by default to be
  *     backwards compatible. Projects are advised to turn this flag always off.
+ * @export
```

Comment:
i believe this is only accessed in this file and doesn't need to be exported

Reply:
`@export` removed